src/pool/rpc.c

```diff
@@ -104,6 +107,24 @@ crt_proc_struct_daos_pool_space(crt_proc_t proc, struct daos_pool_space *ps)
 	return 0;
 }
 
+static int
+crt_proc_struct_daos_space(crt_proc_t proc, struct daos_space *ps)
```

Comment:
As we discussed, you don't have to deal with space usage in this version. In the future, the returned target information will be in daos_target_info_t (probably needs be extended to including fragment information).